packages/aws-cdk/lib/api/aws-auth/credential-plugins.ts

```diff
@@ -151,7 +151,7 @@ function v3ProviderFromV2Credentials(x: SDKv2CompatibleCredentials): AwsCredenti
 function refreshFromPluginProvider(current: AwsCredentialIdentity, producer: () => Promise<PluginProviderResult>): AwsCredentialIdentityProvider {
   return async () => {
     // eslint-disable-next-line no-console
-    console.error(current, Date.now());
+    info(format(current), Date.now());
```

Comment:
you should remove the `// eslint-disable-next-line no-console` on this any any other instances where you no longer call the `console`.

Reply:
There might be a good reason for it: Is the switch from `error` to `info` on purpose?

Reply:
Talked about this: Everything goes to stderr by default in most cases.